src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/OnOffClientFragment.kt

```diff
@@ -85,6 +102,103 @@ class OnOffClientFragment : Fragment() {
     })
   }
 
+  private suspend fun cancelReporting() {
+    val bindingCluster = ChipClusters.BindingCluster(
+      ChipClient.getConnectedDevicePointer(
+        deviceIdEd.text.toString().toLong()
+      ), 1
```

Comment:
could we add constants for the numbers here? the 0, 1, 112233, 0xFFFF and so on? I believe that would help redability.